test/data-ingest/mzcompose.py

```diff
@@ -107,7 +106,9 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
         "mysql",
     )
 
-    executor_classes = [MySqlExecutor, KafkaRoundtripExecutor, KafkaExecutor]
+    # KafkaRoundtripExecutor disabled due to intermittent test failure (https://github.com/MaterializeInc/database-issues/issues/8657)
```

Comment:
If you use a format like `TODO: Reenable when database-issues#8657 is fixed` then I get automatically notified when the issue is closed and the TODO is still in the code